assets/javascripts/tests.js

```diff
@@ -402,6 +402,10 @@ function renderTestLists() {
 
   // add a handler for the actual filtering
   $.fn.dataTable.ext.search.push(function (settings, data, dataIndex) {
+    if (settings.nTable.getAttribute('id') !== 'results') {
```

Comment:
I'm wondering whether this could be improved. This function is still called for every entry of each table. Can it be registered only for the finished table?
I don't know myself right now how to do it, and it's not really a problem, as it returns immediately, but maybe you could have a look?

Reply:
I had a look but didn't yet find a way to register a such a function exclusively to a specific table, maybe `afnFiltering` might work